nevergrad/functions/test_base.py

```diff
@@ -48,6 +48,27 @@ def test_experiment_function() -> None:
     )
 
 
+def test_array_experiment_function() -> None:
+    iarrayfunc = base.ArrayExperimentFunction(lambda x: sum(x),   # type: ignore
+        p.Array(shape=(10,)).set_bounds(-0.5, 6.).set_name(""),
+        symmetry=247,
+    )
+    iarrayfunc2 = base.ArrayExperimentFunction(lambda x: sum(x),   # type: ignore
+        p.Array(shape=(10,)).set_bounds(-0.5, 6.).set_name(""),
+        symmetry=111,
+    )
+    iarrayfunc3 = base.ArrayExperimentFunction(lambda x: sum(x),   # type: ignore
+        p.Array(shape=(10,)).set_bounds(-0.5, 6.).set_name(""),
+        symmetry=111,
+    )
```

Comment:
```suggestion
    iarrayfuncs = [
        base.ArrayExperimentFunction(
            lambda x: sum(x),   # type: ignore
            p.Array(shape=(10,)).set_bounds(-0.5, 6.).set_name(""),
            symmetry=s,
        ) for s in [247, 111, 111]
    ]
```
always avoid duplicated code